commands/deploy/container.py

```diff
@@ -100,3 +101,32 @@ def push(ctx, target, version=None):
     if target == 'service':
         ctx.run(f"docker tag {name}-nginx:{version} {REPOSITORY}/{name}-nginx:{version}", echo=True)
         ctx.run(f"docker push {REPOSITORY}/{name}-nginx:{version}", echo=True, pty=True)
+
+
+@task(help={
+    "target": "Name of the project you want to list images for: service or harvester",
+})
+def print_available_images(ctx, target):
+    # Check the input for validity
+    if target not in TARGETS:
+        raise Exit(f"Unknown target: {target}", code=1)
+
+    # Load info
+    target_info = TARGETS[target]
+    name = target_info["name"]
+
+    # Start boto
+    session = boto3.Session(profile_name="pol-prod")
```

Comment:
is het de bedoeling dat dit altijd prod is?

Reply:
Ja, want onze images gaan altijd naar prod. Ook als we die images deployen naar dev of acc.